src/CoreBundle/private/modules/CMSTableExport/CMSTableExport.class.php

```diff
@@ -145,6 +149,28 @@ public function GenerateExport()
         $this->getDownload($sFileType);
     }
 
+    private function provideExportMemory(): void
+    {
+        $exportMemoryUnits = ServiceLocator::getParameter('chameleon_system.core.export_memory');
+
+        $inBytes = $this->unitToInt($exportMemoryUnits);
+
+        if ($inBytes > 256*1024*1024) {
+            ini_set('memory_limit', $exportMemoryUnits);
+        }
+        // else filter out bogus values
```

Comment:
what is this?

Reply:
description of the if

Reply:
Then we should refactor this like to make it clear. This way it looks like you planed on an else but did not come around to implement it yet:

```php

$exportMemoryUnits = ServiceLocator::getParameter('chameleon_system.core.export_memory');

$inBytes = $this->unitToInt($exportMemoryUnits);

if ($inBytes <= 256*1024*1024) {
    // ignore small values
    return ;
}

ini_set('memory_limit', $exportMemoryUnits);
```

I do not think that makes any sense though. What would make sense here is to check if the value provided is grater than the current memory settings - since it makes no sense to reduce the memory for the export.

Reply:
and "bogus" is anything below 256m... I don't think we can assume that nobody would ever attempt to use less memory...

Reply:
change to check configured against current value

and added correctness check for values